lib/tasks/hydranorth.rake

```diff
@@ -6,15 +6,27 @@ namespace :hydranorth do
     Rails.logger
   end
 
+  desc "set up environment specific rake task loggers to capture STDOUT"
+  task :setup_logger => :environment do
+    logger      = Logger.new(STDOUT)
+    logger.level = Logger::INFO
+    Rails.logger = logger
+  end
+
   desc "Remove lapsed embargoes"
-  task :remove_lapsed_embargoes => :environment do |t|
+  task :remove_lapsed_embargoes => :setup_logger do |t|
     items = Hydra::EmbargoService.assets_with_expired_embargoes
+    logger.info "Number of items that have expired embargoes: #{items.count}"
+    count = 0
     items.each do |item|
+      logger.info "clear expired embargo for #{item.id}"
```

Comment:
how about a timestamp on this line? Then you could grep for the noid and get all the info on one line.